suite-native/module-settings/src/components/ApplicationSettings.tsx

```diff
@@ -32,41 +35,63 @@ export const ApplicationSettings = () => {
     };
 
     return (
-        <SettingsSection title="Application">
+        <SettingsSection title={translate('moduleSettings.items.application.devUtils.title')}>
```

Comment:
```suggestion
        <SettingsSection title={translate('moduleSettings.items.application.title')}>
```

Reply:
If you fix the type for SettingsSection.title, you can then user Translation component. I believe it should be our preferred way.